docs/proposals/00-template.md

```diff
@@ -0,0 +1,20 @@
+# Headline of GEP
+
+## Table of Contents
+
+- [Headline of GEP](#headline-of-gep)
+  - [Table of Contents](#table-of-contents)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+
+## Summary
+
+## Motivation
+
+### Goals
+### Non-Goals
+
+## Proposal
```

Comment:
Should we also add a `Design Details` section with a Plan for graduation and so on like in the kubernetes KEPs?

Reply:
Hm, to my knowledge we didn't really graduate our features as strictly as Kubernetes does it. Do you see the necessity of doing it now?

Reply:
No, not really. As discussed, let's leave it like it is and add it once we see the necessity for it.